src/addForm.js

```diff
@@ -27,7 +27,6 @@ const printForm = () => {
   formContainer.appendChild(submitBtn);
 
   submitBtn.addEventListener('click', () => {
-    imgLoading.classList.remove('d-none');
     submitBtn.disabled = true;
     newUserScore(nameInp.value, scoreInp.value);
     nameInp.value = '';
```

Comment:
- [ ] Score input should be a integer not a string, kindly validate the input to receive only typeof numbers.

![arr6](https://user-images.githubusercontent.com/55635977/131878758-beeb9ee7-c80c-442f-a2c0-3287f4f0d226.png)